CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -175,7 +186,7 @@ async def manual_report(state_file: UploadFile = File(None), cdc_file:  UploadFi
 
 
 @app.post("/automatic_report")
-async def automatic_report(year: int, cdc_file:  UploadFile = File(None)):
+async def automatic_report(year: int, cdc_file:  UploadFile = File(None), isCDCFilter: bool = Form(False)):
     # Run query to retrieve data from NBS ODSE database
     (column_names, state_content) = run_query(year)
```

Comment:
instead of isCDCFilter: bool = Form(False), you should just do - `isCDCFilter: bool`